CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 2.0.1 - March 18, 2020
+- Fixed a bug in `DefaultIlpClient`. "Bearer " prefix was not being prepended to auth tokens, which caused authentication issues on Hermes.
+	- "Bearer " prefix now gets prepended to auth tokens, if it is not already there
```

Comment:
Version bump?

Reply:
This was a diff on a merge to another branch that was out of date with master... I think. The github UI is weird. 

No longer an issue, we're merging to master and the dependent PRs have been merged.